src/redux/reducers/authReducer.ts

```diff
@@ -28,14 +28,24 @@ export const registerReducer = createSlice({
     },
     logUserOut: (state) => {
       state.loading = true;
-      setCookie(null, "userId", "");
-      state.error = false;
-      state.creds.userId = "";
-      state.creds.email = "";
-      state.creds.createdAt = "";
-      state.creds.isVerified = false;
-      state.loading = false;
-    },
+    
+      try {
+        // Clear user ID cookie
+        setCookie(null, "userId", "");
```

Comment:
There is a function called destroy cookies in nookies. You should implement that.